package.json

```diff
@@ -6,6 +6,8 @@
   "author": "Christian Schmidt <cschmidt.solingen@gmail.com>",
   "license": "MIT",
   "scripts": {
+    "build": "webpack",
```

Comment:
please create a css - file. include it + check if the webpack configuration works with css-files too.
did you want to use plain css, sass or less?